src/components/KuadrantDNSPolicyCreatePage.tsx

```diff
@@ -1,6 +1,5 @@
 import * as React from 'react';
 import Helmet from 'react-helmet';
-import yaml from 'js-yaml';
```

Comment:
Can probably drop from package.json now

Reply:
yes, will remove as it's not used elsewhere.
It might get re-introduced with #59